tests/examples/naughty/colors.lua

```diff
@@ -0,0 +1,19 @@
+
+local beautiful = require("beautiful") --DOC_HIDE
+
+local text = [[An <b>important</b>
+<i>notification</i>
+]]
+
+require("naughty").notify {
+    title        = "Hello world!",
+    text         = text,
+    icon         = beautiful.icon,
+    bg           = "#0000ff",
+    fg           = "#ff0000",
+    fond         = "verdana 14",
```

Comment:
typo in `fond` and also Verdana could be easily missing in the system, mb better just Sans?